plugin-flex-ts-template-v2/src/utils/serverless/PhoneNumbers/PhoneNumberService.ts

```diff
@@ -45,13 +45,13 @@ class PhoneNumberService extends ApiService {
 
   private EXPIRY = 86400000; // 1 day
 
-  async getAccountPhoneNumbers(): Promise<ListPhoneNumbersResponse> {
+  async getAccountPhoneNumbers(includeOutgoing: boolean): Promise<ListPhoneNumbersResponse> {
```

Comment:
do we need something to distinguish here whether the cached result includes outgoing or not?  if you have the flag set to false then disable it and vice versa, it should renew the cache.  Perhaps the easiest way to do it is to have two separate cache keys

Reply:
Agreed, done!